src/main/resources/db/migration/0300/V320__AdHocPlotObservations.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE tracking.ad_hoc_observation_types (
```

Comment:
Should these "observation types" only be scoped to the ad hoc plot observations? Or should they just exist at `tracking.observation_types`?